source/shared/cpp/ObjectModel/SharedAdaptiveCard.cpp

```diff
@@ -147,7 +147,11 @@ void AdaptiveCard::_ValidateLanguage(const std::string& language, std::vector<st
 {
     try
     {
-        if (language.empty() || language.length() == 2 || language.length() == 3)
+        if (language.empty())
```

Comment:
from the c++ documentation, leaving the locale empty retrieves the user preferred locale, if we can initialize once the user preferred locale we should only deal with this problem once

https://en.cppreference.com/w/cpp/locale/locale#:~:text=An%20object%20of%20class%20std%3A%3Alocale%20is%20an%20immutable,locale%20object%20is%20associated%20with%20each%20std%3A%3Abasic_regex%20object.